src/style.css

```diff
@@ -0,0 +1,54 @@
+@font-face {
+  font-family: Avenir;
+  src: url(font/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+body {
+  padding-top: 12px;
+}
+
+.search-bar {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
+  padding-left: 62px;
+  margin-bottom: 20px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+  background-image: url(images/Search.svg);
+  color: #3d4e61;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.search-bar.small {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element